Xamarin.Forms.NeoControls/NeoView.xaml.cs

```diff
@@ -186,6 +190,8 @@ protected virtual void DrawInnerShadow(RenderContext context)
         {
             var fShadowDistance = Convert.ToSingle(ShadowDistance);
             var darkShadow = Color.FromRgba(DarkShadowColor.R, DarkShadowColor.G, DarkShadowColor.B, Elevation);
+            var lightShadow = Color.FromRgba(LightShadowColor.R, LightShadowColor.G, LightShadowColor.B, Elevation);
```

Comment:
Interessante, imaginei que i light Shadow precisaria ser sempre solido, pois no neomorfismo temos esse efeito de "continuidade" entre o fundo e o controle. Porém, faz sentido alterar a "força" dessa sombra com relação à elevação.
Seria interessante uma imagem para visualizar o efeito com alteração dessa opacidade.

Reply:
A verdade é que precisamos diferencia-la, porque caso o fundo seja estilo dark, precisamos de mais escuro e menos claro, e caso o fundo seja light, precisamos justamente do contrário, menos intensidade no preto, e mais intensidade no branco

Reply:
Sim, mas para isso está aberta a possibilidade de alterar a cor, podendo utilizar um hexa decimal por exemplo.

O Elevation é a aplicação da opacidade em cima dessa cor ja definida, para dar ao programador uma ferramenta que possa utilizar nesse sentido. Criando efeitos de diferentes relevos ou animações.

Acredito que não seja o caso aplicado ao LightShadow